students/ejackie/session02/fizzbuzz.py

```diff
@@ -0,0 +1,14 @@
+# numbers between 1 to 100 inclusive
```

Comment:
Please use a header in all your files. Should include Author, Date, Version and a brief description